crowbar_framework/lib/crowbar/error/upgrade_status.rb

```diff
@@ -29,8 +29,9 @@ def initialize(step_name = "")
     end
 
     class StartStepOrderError < StandardError
-      def initialize(step_name = "")
-        super("Start of step #{step_name} requested in the wrong order")
+      def initialize(step_name = "", correct_step = "")
+        super("Start of step '#{step_name}' requested in the wrong order. " \
+        "Correct next step is '#{correct_step}'.")
```

Comment:
what if `correct_step` (the default) is empty? the expression wouldn't make sense anymore. I think you need to construct the help string under some conditions